src/test/java/seedu/address/logic/commands/wallet/WalletFindCommandTest.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands.wallet;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.commons.core.Messages.MESSAGE_TRANSACTIONS_LISTED_OVERVIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalWallet.DUCK_RICE;
+import static seedu.address.testutil.TypicalWallet.MRT_CONCESSION;
+import static seedu.address.testutil.TypicalWallet.TA_JOB;
+import static seedu.address.testutil.TypicalWallet.getTypicalWallet;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.transaction.DescriptionContainsKeywordsPredicate;
+
+public class WalletFindCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), getTypicalWallet(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), getTypicalWallet(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        DescriptionContainsKeywordsPredicate firstPredicate =
+                new DescriptionContainsKeywordsPredicate(Collections.singletonList("first"));
+        DescriptionContainsKeywordsPredicate secondPredicate =
```

Comment:
[Question] Did you change the test method in DescriptionContainsKeywordPredicate to use StringUtil.equalsSubWordIgnoreCase?

Reply:
Thank you! Just added it.